flow/libs/intl.js.flow

```diff
@@ -10,6 +9,4 @@ declare type IntlType = {
   NumberFormat: Class<Intl$NumberFormat>,
 }
 
-// Mark it as possibly undefined since
-// Safari and some recent versions of Firefox for Android do not implement it.
-declare var Intl: typeof undefined | IntlType;
+declare var Intl: IntlType;
```

Comment:
I think it's fine to add this as a workaround but ideally we want to say that `Intl` *can* be undefined in some browsers. This will make sure that we don't forget to write defensive code around it.

Can you put a comment in here linking to this Flow bug? I think this is what we're hitting. https://github.com/facebook/flow/issues/4077